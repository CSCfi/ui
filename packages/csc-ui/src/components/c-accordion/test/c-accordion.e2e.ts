import { newE2EPage } from '@stencil/core/testing';

describe('c-accordion', () => {
  it('should do SOMETHING!', async () => {
    const page = await newE2EPage();
    await page.setContent(
      `<c-accordion><c-accordion-item>Lorem ipsum dolor sit amet.</c-accordion-item></c-accordion>`,
    );
    await page.waitForChanges();

    const listElement = await page.find('c-accordion');

    await page.waitForChanges();

    expect(listElement).toMatchSnapshot('JHORMA');
  });
});
