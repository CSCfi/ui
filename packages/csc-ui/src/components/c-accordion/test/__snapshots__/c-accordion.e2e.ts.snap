// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`c-accordion should do SOMETHING! 1`] = `
CDPPage {
  "_e2eClose": [Function],
  "_e2eElements": Array [
    <div class="c-accordion">
  <slot></slot>
</div>,
  ],
  "_e2eEventIds": 0,
  "_e2eEvents": Map {},
  "_e2eGoto": [Function],
  "close": [Function],
  "compareScreenshot": [Function],
  "debugger": [Function],
  "emitter": Object {
    "all": Map {
      "console" => Array [
        [Function],
      ],
      "pageerror" => Array [
        [Function],
      ],
      "requestfailed" => Array [
        [Function],
      ],
      "request" => Array [
        [Function],
      ],
    },
    "emit": [Function],
    "off": [Function],
    "on": [Function],
  },
  "eventsMap": Map {
    "console" => Array [
      [Function],
    ],
    "pageerror" => Array [
      [Function],
    ],
    "requestfailed" => Array [
      [Function],
    ],
    "request" => Array [
      [Function],
    ],
  },
  "find": [Function],
  "findAll": [Function],
  "getDiagnostics": [Function],
  "goto": [Function],
  "setContent": [Function],
  "spyOnEvent": [Function],
  "waitForChanges": [Function],
  "waitForEvent": [Function],
}
`;

exports[`c-accordion should do SOMETHING!: JHORMA 1`] = `
<div class="c-accordion-item__content-wrapper">
  <div aria-labelledby="header__02729a0d-7deb-4832-8b97-7fdcb173261a" class="c-accordion-item__content" id="panel__02729a0d-7deb-4832-8b97-7fdcb173261a" role="region">
    <slot>
      ðŸ¤¡
    </slot>
  </div>
</div>
`;
